pkg/storaged/lvol-tabs.jsx

```diff
@@ -46,6 +57,122 @@ function lvol_rename(lvol) {
     });
 }
 
+const StructureDescription = ({ client, lvol }) => {
+    const vgroup = client.vgroups[lvol.VolumeGroup];
+    const pvs = (vgroup && client.vgroups_pvols[vgroup.path]) || [];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20231026-132112-61c336d9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)